packages/features/bookings/lib/getBookingFields.ts

```diff
@@ -254,6 +254,12 @@ export const ensureBookingInputsHaveSystemFields = ({
       name: "rescheduleReason",
       defaultPlaceholder: "reschedule_placeholder",
       required: false,
+      views: [
+        {
+          id: "reschedule",
+          label: "Reschedule View",
```

Comment:
Having a label allows that label to be shown in UI. This would be used in a followup PR to show rescheduleReason a bit differently